snapcraft/storeapi/__init__.py

```diff
@@ -97,7 +97,8 @@ def __init__(self, conf, root_url):
         self.session.mount('https://', HTTPAdapter(max_retries=5))
 
         self._snapcraft_headers = {
-            'X-SNAPCRAFT-VERSION': snapcraft.__version__
+            'X-SNAPCRAFT-VERSION': snapcraft.__version__,
+            'User-Agent': 'snapcraft/{}'.format(snapcraft.__version__),
```

Comment:
Snapd does `snapd/123 (series 16; classic; testing) ubuntu/16.10 (amd64) linux/4.8.0-3-generic`
IFAICS, snapcraft does not depend on kernels (am I wrong ?), so we could start with snapcraft and host entities: `snapcraft/123 (classic; testing) ubuntu/16.10 (amd64)`